### ISSUE TITLE
Q: Refactor Picon Path Calculation

### ISSUE THREAD

**Issue by jbleyel** [openatv/enigma2#3126]
The current picon search function is based on all mounted devices with a picon subfolder and in addition a lcdpicon subfolder.
That means all of these folders will be scanned for each picon all the time.

My plan is to remove that completely.
The new search function will only search in one folder and this folder can be defined as setting.

Any objections?

**Comment by Papi2000-new** [comment]
Yes, this sounds pretty good (but the skinner will have once again another construction set).
But in fact, that can be the future. Maybe, in combination with picon_lcd for separate picons for lcd (better contrast in b/w)?

**Comment by wedebe** [comment]
Sounds good.

Ideally, the `Picon install path` Plugin Manager setting would be moved to... `Channel Selection Settings`?

`Picon` path could be set to `Automatic`, which would - on bootup - scan/probe mounts and save the found path as a variable to be called from a `getPiconFolder` (or `getPiconDirectory`)

*OR*... On bootup, mount the found picon folder if not /picon (eg. /media/mmc/picon) to /picon, such that /picon is **always** available.

**Comment by jbleyel** [comment]
Move `Picon install path` to `Channel Selection Settings` is wrong because this is the wrong context.

**Comment by AbuBaniaz** [comment]
I hope you allow users to have one folder for LCD picons and another for the osd ones.

**Comment by jbleyel** [comment]
> I hope you allow users to have one folder for LCD picons and another for the osd ones.

Yes. That’s one of reasons why i wann refactor.
